api/envoy/api/v2/auth/cert.proto

```diff
@@ -73,7 +73,7 @@ message TlsCertificate {
   // The TLS private key.
   core.DataSource private_key = 2;
 
-  // [#not-implemented-hide:]
+  // The password to decrypt the TLS private key.
```

Comment:
Can you specify that if this is empty we assume the key has no password? (Or something to that effect)

Reply:
Addressed via 9595292.